### DISCUSSION TITLE
v3.10.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#9651] [Releases & Announcements]
## Changelog

- Linear Integration (alpha)
- Gmail channel
- Improved reconnection logic
- Performance Improvements to reporting APIs
- FCM API changes
- Help centre improvements ( support for linebreaks, tables etc, in the API )
- Ability to configure email channel settings in super admin
- Numerous bug fixes and enhancements

###
Thanks to  @fv3rdugo, @s00d , @ajnadox, @eltociear, @abehnamfard for the contributions

-----

Notes:

We were using FCM legacy APIs to send push notifications to Android and iOS devices. As these APIs were deprecated on June 20, 2023, and will be removed in June 2024, we have added support for HTTP v1 API notifications in the latest version.  Users of Chatwoot cloud and self-hosted instances using the Chatwoot relay server don't need to make any changes. If you are using the mobile app, ensure that you have installed the latest Chatwoot mobile app (1.10.41).

However, for users with a self-hosted server with  **custom-branded mobile app** some changes are necessary. You can learn more about these changes here: https://firebase.google.com/docs/cloud-messaging/migrate-v1 

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.10.0'>v3.10.0</a>.</em>

**Comment by jochenstu** [comment]
Can you please expand on the FCM changes required with regards to self-hosted instanced? The provided link does not make immediately obvious how these changes need to be applied in regards to Chatwoot.

**Reply by muhsin-k** [reply]
@jochenstu If you are using the official mobile app, push notifications should work without any issues. Just ensure that the environment variable `ENABLE_PUSH_RELAY_SERVER` is set to true. Changes are only necessary for users who use a **custom-branded** mobile app.

**Reply by jochenstu** [reply]
Re-reading the initial posting I am still not entirely able to get to this point, so the clarification was definitely helpful. Thanks!

**Reply by muhsin-k** [reply]
Sorry for the confusion. I've updated the initial post. Thank you.

**Reply by avinash-marsdevs** [reply]
I am not able to find 1.10.41. i can only see https://github.com/chatwoot/chatwoot-mobile-app/releases/tag/1.10.26 as latest.

**Reply by muhsin-k** [reply]
https://github.com/chatwoot/chatwoot-mobile-app/releases/tag/v1.0.41

**Reply by avinash-marsdevs** [reply]
Also in the version we should be using FIREBASE_PROJECT_ID and FIREBASE_CREDENTIALS in the env which is missing in the .env.example and docs for this change? Correct me if I am wrong

**Reply by avinash-marsdevs** [reply]
> https://github.com/chatwoot/chatwoot-mobile-app/releases/tag/v1.0.41

Sorry again this is 1.0.41. I can find this in the tags. but Release notes mentioned with 1.10.41 @muhsin-k

**Reply by muhsin-k** [reply]
@avinash-marsdevs https://github.com/chatwoot/chatwoot-mobile-app/releases/tag/v1.10.41

**Comment by jordan-brough** [comment]
> Ability to configure email channel settings in super admin

Where can I find more info about this?

**Reply by sojan-official** [reply]
https://www.chatwoot.com/docs/self-hosted/configuration/features/email-channel/google-workspace-setup/
